src/components/Text/Text.tsx

```diff
@@ -10,6 +10,7 @@ export interface TextProps extends TextBaseProps, ColorTextBaseProps {
     style?: React.CSSProperties;
     className?: string;
     children?: React.ReactNode;
+    title?: string;
```

Comment:
I think it is better to name it `titleAttribute`, because in some components we use `title` property for text rendering

Reply:
[0ccd469](https://github.com/gravity-ui/uikit/pull/341/commits/0ccd4699fea51ffbdb4f2118da6f3467ff145b3d)